pyproject.toml

```diff
@@ -76,3 +76,7 @@ distance-dirty = "{base_version}+{distance}.{vcs}{rev}.dirty"
 method = "git" 
 match = ["*"]
 default-tag = "0.0.0"
+
+
+#[project.entry-points.pytest11]
```

Comment:
Do we plan on uncommentating these?

Reply:
No, I'll cut them now. They were an experiment for exposing some of our fixtures via entrypoints so we could use them for e.g. `alchemiscale_fah`, but I did this in a different way instead.